src/lib/kube_job.ml

```diff
@@ -90,6 +85,11 @@ let command_must_succeed_with_output ~log ?additional_json job cmd =
 let start ~log t =
   let spec = t.specification in
   let open Specification in
+  let secret_name f =
+    String.take
+      (t.id ^ (Digest.string (File_contents_mount.show f) |> Digest.to_hex))
+      55 (* The max is 63 + we want to add "-volume" *)
```

Comment:
Got is, so 36 characters, leaving 19 for the mount digest. That should be fine :)